sqldb/sqlc/migrations/000005_payments.up.sql

```diff
@@ -0,0 +1,284 @@
+-- payments table contains all the main information for a payment.
+CREATE TABLE IF NOT EXISTS payments (
+    -- The id of the mpp payment.
+    id BIGINT PRIMARY KEY,
+
+    -- payment_status is the status of the payment.
+    payment_status INTEGER NOT NULL REFERENCES payment_status_types(id) ON DELETE CASCADE,
+
+    -- payment_type is the type of the payment.
+    payment_type INTEGER NOT NULL REFERENCES payment_types(id) ON DELETE CASCADE,
+
+    -- amount_msat is the amount of the payment in milli-satoshis.
+    amount_msat BIGINT NOT NULL,
+
+    -- payment_request is the payment request of the payment.
+    payment_request BLOB UNIQUE NOT NULL,
+
+    -- created_at is the timestamp of the payment.
+    created_at TIMESTAMP NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS payments_payment_status_idx ON payments(payment_status);
+CREATE INDEX IF NOT EXISTS payments_payment_type_idx ON payments(payment_type);
+CREATE INDEX IF NOT EXISTS payments_created_at_idx ON payments(created_at);
+
+
+-- payment_status_types stores the different types of a mpp payment.
+CREATE TABLE IF NOT EXISTS payment_status_types(
+    -- id is the id of the payment status type.
+    id INTEGER PRIMARY KEY,
+
+    -- description is the description of the payment status type.
+    description TEXT NOT NULL
+);
+
+
+-- invoice_event_types defines the different types of invoice events.
+INSERT INTO payment_status_types (id, description) 
```

Comment:
Do these status values capture the latest fixes/wisdom w.r.t edge cases we've seen in the wild with payments in "stuck" states?